### ISSUE TITLE
In Google Chrome, NVDA does not read an error message when an incorrect value is entered

### ISSUE THREAD

**Issue by RD-Brown** [nvaccess/nvda#8657]
When an error message is generated in Google Chrome, the user must press Enter two times before the message is read by NVDA.  (TestNav, Pearson’s test delivery platform, is used to demonstrate this error inside Google Chrome.)

### Steps to reproduce:
1.	In Chrome, go to the link:  https://ref.testnav.com/client/index.html
2.	For the Username enter “LGN123626755”
3.	For the password enter “TGKCBMGN,” and press Enter.
4.	Press “B” until you reach the Start button, and press Enter.
5.	Press “E,” and you will be prompted, “Enter seal code.”
6.	Press Enter, type “1233,” and press Enter.  
7.	Note that although, “Incorrect seal code,” is displayed, NVDA does not read the message.  Press Enter a second time, and note that this time the message is read.

### Actual behavior:
Error message is not read until Enter is  pressed twice.

### Expected behavior:
Error message should be read the first time Enter is  pressed.

#### NVDA Installed/portable/running from source:
Installed

#### NVDA version:
2018.2

#### Windows version:
Windows 10 Version 10.0.15063 Build 15063

#### Name and version of other software in use when reproducing the issue:
Google Chrome V68.0.3440.106 (Official Build) (64-bit)

#### Does the issue still occur after restarting your PC?
Yes

#### Have you tried any other versions of NVDA?
No

**Comment by jscholes** [comment]
Are you the designer or developer of this page?  If so: the div with ID of `startSectionError` has a CSS class of `hidden` applied until the error message is displayed.  In the JavaScript for the page, are you removing that CSS class before or after updating the inner HTML of the div with the text of the error message?  If the inner HTML is updated while the div is still hidden, the change will not be spoken by a screen reader.  AFAIK, changes to an ARIA live region must take place while the region is visible to the screen reader in the DOM of the page in order for the change to be spoken.

If you're a user of this website rather than a developer, could you pass these comments along to the page designers and see what they say, if at all possible?  I suspect this is not an NVDA issue, but a problem with this page specifically.

P.S. this is unrelated to the issue at hand, but this website seems to grab a lot of keystrokes (e.g. Alt+D to jump to the address bar), preventing them from being usable by a screen reader user.

**Comment by Adriani90** [comment]
@RD-Brown unfortunately the test account does not work anymore so I am not able to reproduce this issue anymore. Could you please give us an update if this is still reproducible with NVDA 2019.1.1 on Chrome 74?

**Comment by Adriani90** [comment]
We do not have an update from the author since more than one year. We cannot reproduce this issue from the current description. I also think this could be a duplicate of #8318. Closing for now as abandoned. Please comment on it if you still have this issue. Thanks.
